cirq-core/cirq/ops/phased_x_z_gate_test.py

```diff
@@ -313,3 +313,39 @@ def test_str_diagram() -> None:
 0: ───PhXZ(a=0.125,x=0.5,z=0.25)───
     """,
     )
+
+
+@pytest.mark.parametrize(
+    'clifford_gate',
+    [g.to_phased_xz_gate() for g in cirq.SingleQubitCliffordGate.all_single_qubit_cliffords],
+)
+def test_has_stabilizer_effect_true_for_cliffords(clifford_gate):
+    assert cirq.has_stabilizer_effect(clifford_gate)
+
+
+@pytest.mark.parametrize(
+    'clifford_gate',
+    [g.to_phased_xz_gate() for g in cirq.SingleQubitCliffordGate.all_single_qubit_cliffords],
+)
+@pytest.mark.parametrize('global_phase', 1j ** np.random.default_rng(42).random(10))
```

Comment:
nit - we may sample all complex quadrants here.  There is no need to use fixed seed, `PhasedXZGate.from_matrix` ignores the global phase and recovers back the `clifford_gate`.

```suggestion
@pytest.mark.parametrize('global_phase', 1j ** np.random.uniform(0, 4, 10))
```